src/logic/tokens/utils/formatAmount.ts

```diff
@@ -3,47 +3,53 @@
-const lt1kFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 5 })
-const lt10kFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 4 })
-const lt100kFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 3 })
-const lt1mFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 2 })
-const lt10mFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 1 })
-const lt100mFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 0 })
+const LOCALE = []
+
+const lt1kFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 5 })
+const lt10kFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 4 })
+const lt100kFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 3 })
+const lt1mFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 2 })
+const lt10mFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 1 })
+const lt100mFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 0 })
 // same format for billions and trillions
-const lt1000tFormatter = new Intl.NumberFormat([], { maximumFractionDigits: 3, notation: 'compact' } as any)
+const lt1000tFormatter = new Intl.NumberFormat(LOCALE, { maximumFractionDigits: 3, notation: 'compact' })
+const gt1000tFormatter = new Intl.NumberFormat(LOCALE, { notation: 'compact' })
 
 export const formatAmount = (number: string): string => {
   let numberFloat: number | string = parseFloat(number)
 
   if (numberFloat === 0) {
     numberFloat = '0'
   } else if (numberFloat < 0.001) {
-    numberFloat = '< 0.001'
-  } else if (numberFloat < 1000) {
+    numberFloat = `< ${lt1kFormatter.format(0.001)}`
+  } else if (numberFloat < 1_000) {
     numberFloat = lt1kFormatter.format(numberFloat)
-  } else if (numberFloat < 10000) {
+  } else if (numberFloat < 10_000) {
     numberFloat = lt10kFormatter.format(numberFloat)
-  } else if (numberFloat < 100000) {
+  } else if (numberFloat < 100_000) {
     numberFloat = lt100kFormatter.format(numberFloat)
-  } else if (numberFloat < 1000000) {
+  } else if (numberFloat < 1_000_000) {
     numberFloat = lt1mFormatter.format(numberFloat)
-  } else if (numberFloat < 10000000) {
+  } else if (numberFloat < 10_000_000) {
     numberFloat = lt10mFormatter.format(numberFloat)
-  } else if (numberFloat < 100000000) {
+  } else if (numberFloat < 100_000_000) {
     numberFloat = lt100mFormatter.format(numberFloat)
   } else if (numberFloat < 10 ** 15) {
     numberFloat = lt1000tFormatter.format(numberFloat)
   } else {
-    numberFloat = '> 1000T'
+    // Localized '> 1000T'
+    numberFloat = `> ${gt1000tFormatter.format(10 ** 15)}`
   }
 
   return numberFloat
 }
 
-const options = { style: 'currency', currency: 'USD', minimumFractionDigits: 2, maximumFractionDigits: 8 }
-const usNumberFormatter = new Intl.NumberFormat('en-US', options)
+const currencyFormatter = new Intl.NumberFormat(LOCALE, {
+  minimumFractionDigits: 2,
+  maximumFractionDigits: 8,
+})
```

Comment:
This needs to be outside of the function for tests to pass.

Reply:
By no longer using a 'currency' style anymore, it now allows for us to use token symbols as well.